### PR TITLE
Ys/planning kampaigns

### PR DESCRIPTION
**Pull request by yoshson**
Allow planning overview for campaigns and some external planning fixes

### CODE REVIEW

File: workbench/planning/reporting.py [hunk 1]
```diff
@@ -790,6 +791,64 @@ def _customer(planned, logged):
     return ret
 
 
+def campaign_planning_external(campaign):
+    return campaign_planning(campaign, True)
```
Comment (matthiask) [inline workbench/planning/reporting.py]:
```suggestion
    return campaign_planning(campaign, external_view=True)
```

File: workbench/planning/reporting.py [hunk 2]
```diff
@@ -790,6 +791,64 @@ def _customer(planned, logged):
...
+
+
+def campaign_planning(campaign, external_view=False):
```
Comment (matthiask) [inline workbench/planning/reporting.py]:
```suggestion
def campaign_planning(campaign, *, external_view=False):
```

**Review by matthiask — COMMENTED** [review]
Sieht super aus!

**Review by yoshson — COMMENTED** [review]
Cool, danke!

**Comment by matthiask** [comment]
Merci dir!

**Pull request merged**